### ISSUE TITLE
LP does not check the right file!

### ISSUE THREAD

**Issue by fblanqui** [Deducteam/lambdapi#235]
Take a file f.dk that does not compile. Add now a file f.lp that compiles. Then, LP will say that f.dk compiles! Example:

183.dk:
```
type : Type.
def Term : type -> Type.

Prod : (A : type) -> (Term A -> type) -> type.
[A,B] Term (Prod A B) --> (x: Term A) -> Term (B x).

Sigma : (A : type) -> (Term A -> type) -> Type.

pair : (A : type) -> (B : (Term A -> type)) -> (x : Term A) -> Term (B x) -> Sigma A (x => B _).

def fst : (A : type) -> (B : (Term A -> type)) -> Sigma A (x => B x) -> Term A.
[B, B', x] fst _ B (pair _ B' x _) --> x.
```

183.lp:
```
symbol Type : TYPE
symbol Term : Type ⇒ TYPE
symbol prod : ∀A, (Term A ⇒ Type) ⇒ Type

rule Term (prod &A &B) → ∀x:Term &A, Term (&B x)

symbol injective Sigma : ∀A, (Term A ⇒ Type) ⇒ TYPE
symbol pair : ∀A, ∀B, ∀x, Term (B x) ⇒ Sigma A B
symbol fst : ∀A, ∀B, Sigma A B ⇒ Term A

rule fst _ _ (pair _ _ &x _) → &x

```

**Comment by rlepigre** [comment]
Fixed by b9285af2803b66c5834548a1f1b3ec13f9897bc4.

Note that we should not allow files `path/name.lp` and `path/name.dk` in the same directory (i.e., under the same module path). If that was the case, then the `path/name.lp` would always be preferred and the `path/name.dk` file ignored (as @fblanqui witnessed).

Lambdapi now fails with the following message with your examples.
```
Both [183.lp] and [183.dk] exist.
```

Note that you could still make the two files cohabit by moving `183.dk` to `legacy/183.dk`, or simply changing its name (and hence module path).

**Comment by fblanqui** [comment]
Thanks Rodolphe!
